src/Security/ScreenAuthenticator.php

```diff
@@ -153,7 +153,7 @@ public function unbindScreen(Screen $screen): void
     private function generateBindKey(): string
     {
         $length = 8;
-        $chars = '123456789ABCDEFGHIJKLMNPQRSTUVWXYZ';
```

Comment:
Why this change?

Reply:
Having just numbers vs. numbers and chars makes no real difference from a security perspective. But having (capital) letters in the code is an inconvenience when entering the code.